compiler/lib/src/main/scala/codegen/ComputeGeneratedFiles.scala

```diff
@@ -10,7 +10,12 @@ object ComputeGeneratedFiles {
   /** Computes autocoded files (XML, C++ and JSON Dictionary) */
   def getAutocodeFiles(tul: List[Ast.TransUnit]): Result.Result[List[String]] =
     for {
+      // Perform analysis only up to the point we need
       a <- enterSymbols(tul)
+      a <- CheckUses.visitList(a, tul, CheckUses.transUnit)
```

Comment:
Why is this analysis added here? It looks like nothing else has changed in this file.

Reply:
It looks like it's needed in ComputeAutocodeCppFiles